File: Javascript/1_Bubble Sort/Bubble Sort.js
function bubbleSort(array){
	for(var i = 0; i < array.length - 1; i++){
		for(var j = 0; j < array.length - 1 - i; j++){
			if(array[j] > array[j+1]){
				var temp = array[j];
				array[j] = array[j+1];
				array[j+1] = temp;
			}
		}
	}
}

var array = [14, 46, 43, 27, 57, 41, 45, 21, 70];
bubbleSort(array);
console.log(array);

File: C#/1_Bubble Sort/Bubble Sort.cs
using System;

namespace Algorithm {
    
	class BubbleSort {
		
		public static void bubbleSort(int[] arr){
			bool swapped = true;

			for(int i = 0; i < arr.Length - 1; i++){
				for(int j = 0; j < arr.Length - i - 1; j++){
					if(arr[j] > arr[j+1]){
						int temp = arr[j];
						arr[j] = arr[j+1];
						arr[j+1] = temp;
						swapped = true;
					}
				}

				if(swapped == false){
					break;
				}
			}
		}

		public static void printArray(int[] arr){
			for(int i = 0; i < arr.Length; i++){
				Console.Write(arr[i] + " ");
			}
		}

		public static void Main(string[] args){
			int[] arr = {5, 2, 42, 6, 1, 3, 2};
			bubbleSort(arr);
			printArray(arr);
		}
	}
}